force-app/main/default/lwc/listViewSelector/listViewSelector.html

```diff
@@ -0,0 +1,38 @@
+<template>
+    <lightning-layout>
+        <lightning-layout-item>
+            <lightning-button-menu
+                variant="container"
+                onselect={handleListViewSelected}
+                label={selectedListView.label}
+                class="listViewSelector"
+                if:true={options}
+            >
+                <div
+                    role="presentation"
+                    class="slds-text-title_caps slds-var-m-around_medium"
+                >
+                    {labels.listViewLabel}
+                </div>
+                <template for:each={options} for:item="option">
+                    <lightning-menu-item
+                        label={option.label}
+                        value={option.value}
+                        key={option.value}
+                        checked={option.isChecked}
+                    ></lightning-menu-item>
+                </template>
+            </lightning-button-menu>
+        </lightning-layout-item>
+        <lightning-layout-item padding="horizontal-small">
+            <lightning-button-icon-stateful
+                icon-name="utility:pin"
+                alternative-text={labels.pinListView}
```

Comment:
@jjbennett , Can we add a hover text too so the users can see it just like the standard list view.

Reply:
Done, thanks for the suggestion!